backend/src/common/services/jwt/jwt.service.ts

```diff
@@ -0,0 +1,22 @@
+import jwt from 'jsonwebtoken';
+import { type JwtPayload } from 'jsonwebtoken';
+
+class JwtService {
+    private readonly secretKey: string;
+
+    public constructor(secretKey: string) {
+        this.secretKey = secretKey;
+    }
+
+    public createToken(payload: unknown): string {
```

Comment:
Why `payload` is unknown? In the task description it is said that _The token should contain the userId._ So, you can just specify `payload` as string type or create a separate JwtPayload type in case it will be changed in the future. What do you think about it?

Reply:
I set the type to unknown for possible further changes, it's short and flexible at the same time. If we want to add something besides userId. But I will think about your suggestion